examples/comparison_ccl_nc_theory_backend.ipynb

```diff
@@ -0,0 +1,232 @@
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "3d21e093-2a42-4e52-9186-addae94e0fc9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ['CLMM_MODELING_BACKEND'] = 'ccl' \n",
+    "\n",
+    "import clmm\n",
+    "from clmm import Cosmology"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "1b975dae-ef11-4931-aa3c-26c99ea421b7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'1.1.8'"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "clmm.__version__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "396abe98-2af2-47fe-b9cc-42b86129850c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cosmo = Cosmology(H0=70.0, Omega_dm0=0.27-0.045, Omega_b0=0.045, Omega_k0=0.0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7e7e1772-3e18-4b85-94a1-f9d071f9f553",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "moo = clmm.Modeling(massdef='mean', delta_mdef=200, halo_profile_model='nfw', )\n",
+    "\n",
+    "moo.set_cosmo(cosmo)\n",
+    "moo.set_concentration(4)\n",
+    "moo.set_mass(1.e15)\n",
+    "\n",
+    "z_cl = 1.\n",
+    "\n",
+    "#source properties\n",
+    "z_source = 2. #all sources in the same plane\n",
+    "z_distrib_func = clmm.utils._chang_z_distrib #sources redshift following a distribution\n",
+    "alpha = [2, -0.5] "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "94e0740b-dfae-41ed-aa3b-f9885be4280f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r3d = np.logspace(2, 2, 100)\n",
+    "rho = moo.eval_3d_density(r3d, z_cl)\n",
+    "Sigma = moo.eval_surface_density(r3d, z_cl)\n",
+    "DeltaSigma = moo.eval_excess_surface_density(r3d, z_cl)\n",
+    "Sigmac = moo.eval_critical_surface_density(z_cl, z_source)\n",
+    "gammat = moo.eval_tangential_shear(r3d, z_cl, z_source)\n",
+    "kappa = moo.eval_convergence(r3d, z_cl, z_source)\n",
+    "\n",
+    "gt_nc = moo.eval_reduced_tangential_shear(r3d, z_cl, z_source)\n",
+    "#Lensing quantities assuming sources follow a given redshift distribution.\n",
+    "gt_z_nc = moo.eval_reduced_tangential_shear(r3d, z_cl, z_source)\n",
+    "\n",
+    "mu_nc = moo.eval_magnification(r3d, z_cl, z_source)\n",
+    "mu_bias_nc = moo.eval_magnification_bias(r3d, z_cl, z_source, alpha)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "4dbd1763-832e-4a67-a496-140f6b315e49",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#del os.environ['CLMM_MODELING_BACKEND']\n",
+    "\n",
+    "#os.environ['CLMM_MODELING_BACKEND'] = 'ccl' \n",
+    "\n"
```

Comment:
I was trying to use both backends to compare the results from NumCosmo and CCL, but I can't find a way to use both in the same notebook. So I ran the notebook for numcosmo, wrote down the results in the last cell, and then changed the backend to ccl.